data_exploration/Perturbseq/curated/json/adamson_2016_pilot_curated.json

```diff
@@ -0,0 +1,187 @@
+{
```

Comment:
Just thinking out loud: since the files in this directory are generated by running the corresponding Jupyter notebooks, normally we wouldn't put them into the repository (because these are output data, not source code).

However, since the files are small and not numerous, it actually does make some sense to keep them in the repo.

There are no actions to be done right now, I'll just leave this comment open until final review iterations and then we can decide (we can also discuss in person)